packages/components/stories/AppHeaderBar.js

```diff
@@ -54,186 +55,140 @@ const typeaheadItems = [
 				description: 'description: Gradu quos cedentium sunt appeterent ita ancoralia instar luna sunt etiam ubi incendente nihil observabant.',
 			},
 		],
-	}];
+	},
+];
 
-const inputForm = {
-	forms: [{
-		form: {
-			pullRight: true,
-			onSubmit: action('onSubmit'),
-			role: 'search',
+const props = {
+	brand: {
+		id: 'header-brand',
+		name: 'Example App Name',
+		onClick: action('onApplicationNameClick'),
+	},
+	logo: {
+		id: 'header-logo',
+		onClick: action('onLogoClick'),
+	},
+	search: {
+		icon: {
+			name: 'talend-search',
+			title: 'icon',
+			bsStyle: 'link',
 		},
-		formgroups: [
+		id: 'header-search',
+		onToggle: action('onSearchClick'),
+	},
+	help: {
+		id: 'header-help',
+		onClick: action('onHelpClick'),
+	},
+	user: {
+		id: 'header-user',
+		items: [
 			{
-				formgroup: {
-					controlId: 'globalSearch',
-					pullRight: true,
-				},
-				formcontrol: {
-					type: 'text',
-					placeholder: 'search anything',
-					onChange: action('onChange in search'),
-				},
+				icon: 'talend-cog',
+				label: 'Settings',
+				onClick: action('onSettingsClick'),
+			},
+		],
+		name: 'User NAME',
+	},
+	products: {
+		id: 'header-products',
+		items: [
+			{
+				icon: 'talend-logo-square',
+				label: 'Portal',
+				onClick: action('onPortalClick'),
 			},
 		],
-		button: {
-			onClick: action('onClick form'),
-			bsStyle: 'link',
-		},
-		buttonLabel: 'submit',
-	}],
-};
-
-const props = {
-	app: 'Example App Name',
-	brandLink: {
-		onClick: action('brandonClick'),
 	},
-	content: [
-		{
-			navs: [
-				{
-					navItems: [
-						{
-							type: 'navItem',
-							item: {
-								icon: 'fa fa-bars',
-								name: 'hello',
-								onClick: action('onClick bars'),
-							},
-						},
-					],
-				},
-				{
-					nav: { pullRight: true },
-					navItems: [
-						{
-							type: 'navItem',
-							item: {
-								icon: 'fa fa-heart',
-								name: 'world',
-								onClick: action('onClick heart'),
-							},
-						},
-						{
-							type: 'dropdown',
-							item: {
-								dropdown: {
-									id: 'myDropdown',
-									title: 'user 1',
-									onSelect: action('dropdown onSelect'),
-								},
-								items: [
-									{
-										icon: 'fa fa-fw fa-cog',
-										name: 'settings',
-										onClick: action('onClick settings'),
-									},
-								],
-							},
-						},
-					],
-				},
-			],
-		},
-	],
 };
 
 const decoratedStories = storiesOf('AppHeaderBar', module)
 	.addDecorator(story => (
 		<div>
 			{story()}
 			<div className="container" style={{ paddingTop: 40 }} />
+			<IconsProvider />
 		</div>
 	));
 
 decoratedStories
 	.addWithInfo('default', () => {
-		if (props.content[1]) {
-			delete props.content[1];
-		}
-		return <AppHeaderBar {...props} />;
+		const headerProps = Immutable.fromJS(props).toJS();
```

Comment:
It was used in List story (https://github.com/Talend/ui/blob/master/packages/components/stories/List.js#L307) so I just reuse it. But we can discuss it I think